library/IvozProvider/Klear/Auth/Adapter.php

```diff
@@ -22,22 +22,18 @@ class Adapter implements \Klear_Auth_Adapter_KlearAuthInterface
 
     public function __construct(\Zend_Controller_Request_Abstract $request, \Klear_Model_ConfigParser $authConfig = null)
     {
-
-        if (!$authConfig->getProperty("brandId")) {
-            throw new \Klear_Exception_Default('brandId not on authConfig');
-        }
-
         $this->_username = $request->getPost('username', '');
         $this->_password = $request->getPost('password', '');
         $brandMapper = new BrandsMapper();
-        $this->_brand = $brandMapper->find($authConfig->getProperty("brandId"));
+        if ($authConfig->getProperty("brandId")) {
```

Comment:
It seems to be an impossible scenario, but the default value is forced at interface level. I'll add a security check just in case